lib/taurus/qt/qtgui/table/taurusgrid.py

```diff
@@ -894,23 +906,11 @@ def build_widgets(self, values, show_labels=False, width=240, height=20, value_w
                     self._widgets_list.append(synoptic_value)
                     count += 1
 
-                # Done in this way as TauValue.mousePressEvent are never called
-                def mousePressEvent(event, obj):
-                    # print 'In cell clicked'
-                    targets = set(str(child.getModelName()) for child in obj.children()
-                                  if hasattr(child, 'underMouse') and child.underMouse() and hasattr(child, 'getModelName'))
-                    [obj.itemClicked.emit(t)
-                     for t in targets]
-
-                cell_frame.mousePressEvent = partial(
-                    mousePressEvent, obj=cell_frame)
-                cell_frame.itemClicked.connect(self.itemClicked)
-
                 widgets_row.append(cell_frame)
             widgets_matrix.append(widgets_row)
         return widgets_matrix
 
-    def itemClicked(self, item_name):
+    def itemClickedHook(self, item_name):
```

Comment:
Not important, but for consistency with other parts of Taurus, could you consider renaming this to `onItemClicked`?
...or even better, make this private by calling it `_onItemClicked`